src/as-libxml.c

```diff
@@ -320,6 +320,7 @@ alloc_doc(Options *opts) {
             }
             opts->line_number_attr = xmlDictLookup(doc->dict, BAD_CAST opts->line_number_attr, -1);
         }
+	doc->encoding = xmlStrdup(BAD_CAST "UTF-8");
```

Comment:
Looks like this is a tab character instead of 8 spaces.. sorry.

Reply:
fixed